src/pages/instances/forms/DuplicateInstanceForm.tsx

```diff
@@ -0,0 +1,225 @@
+            `Instance creation for "${values.instanceName}" failed.`,
+            e,
+          );
+          close();
+        });
+    },
+  });
+
+  return (
+    <Modal
+      close={close}
+      className="duplicate-instances-modal"
+      title="Duplicate Instance"
+      buttonRow={
+        <>
+          <Button
+            appearance="base"
+            className="u-no-margin--bottom"
+            type="button"
+            onClick={close}
+          >
+            Cancel
+          </Button>
+          <ActionButton
+            appearance="positive"
+            className="u-no-margin--bottom"
+            loading={formik.isSubmitting}
+            disabled={
+              !formik.isValid ||
+              projectsLoading ||
+              storagePoolsLoading ||
+              clusterMembersLoading
+            }
+            onClick={() => void formik.submitForm()}
+          >
+            Duplicate and start
+          </ActionButton>
+        </>
+      }
+    >
+      <Form onSubmit={formik.handleSubmit}>
+        <Input
+          {...formik.getFieldProps("instanceName")}
+          type="text"
+          label="New instance name"
+        />
+        {isClustered ?? (
+          <Select
+            {...formik.getFieldProps("targetClusterMember")}
+            id="targetClusterMember"
+            label="Target cluster member"
+            options={clusterMembers.map((clusterMember) => {
+              return {
+                label: clusterMember.server_name,
+                value: clusterMember.server_name,
+              };
+            })}
+          />
+        )}
+        <Select
+          {...formik.getFieldProps("targetStoragePool")}
+          id="storagePool"
+          label="Storage pool"
+          options={storagePools.map((storagePool) => {
+            return {
+              label: storagePool.name,
+              value: storagePool.name,
+            };
+          })}
+        />
+        <Select
+          {...formik.getFieldProps("targetProject")}
+          id="project"
+          label="Target project"
+          options={projects.map((project) => {
+            return {
+              label: project.name,
+              value: project.name,
+            };
+          })}
+        />
+        <Input
+          {...formik.getFieldProps("allowInconsistent")}
+          type="checkbox"
+          label="Ignore copy errors for volatile files"
+          error={
+            formik.touched.allowInconsistent
+              ? formik.errors.allowInconsistent
+              : null
+          }
+        />
+        <Input
+          {...formik.getFieldProps("instanceOnly")}
+          type="checkbox"
+          label="Copy instance without its snapshots"
+          error={
+            formik.touched.instanceOnly ? formik.errors.instanceOnly : null
+          }
+        />
+        <Input
+          {...formik.getFieldProps("stateful")}
+          type="checkbox"
+          label="Copy stateless"
```

Comment:
```suggestion
          label="Copy stateful"
```
The copy is wrong here.